nGram deprecated in elastic4s v7.9.2

Hello,

The following warning appears when running elastic4s v7.9.2:
```
server[ERROR] WARNING: request [PUT http://localhost:9200/XXXXXX] returned 1 warnings: [299 Elasticsearch-7.8.1-b5ca9c58fb664ca8bf9e4057fc229b3396bf3a89 "The [nGram] tokenizer name is deprecated and will be removed in a future version. Please change the tokenizer name to [ngram] instead."]
```

It seems `nGram` should be changed to `ngram` in Tokenizer.scala line 87

Would you be able to make a pr?

On Tue, 1 Dec 2020, 06:05 Vladimir Bodnartchouk, <notifications@github.com>
wrote:

> Hello,
>
> The following warning appears when running elastic4s v7.9.2:
>
> server[ERROR] WARNING: request [PUT http://localhost:9200/XXXXXX] returned 1 warnings: [299 Elasticsearch-7.8.1-b5ca9c58fb664ca8bf9e4057fc229b3396bf3a89 "The [nGram] tokenizer name is deprecated and will be removed in a future version. Please change the tokenizer name to [ngram] instead."]
>
> It seems nGram should be changed to ngram in Tokenizer.scala line 87
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sksamuel/elastic4s/issues/2262>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFVSGQNUBRMBPXDBBAPP7LSSTLZJANCNFSM4UI54YUQ>
> .
>

--- a/elastic4s-core/src/main/scala/com/sksamuel/elastic4s/analysis/Tokenizer.scala
+++ b/elastic4s-core/src/main/scala/com/sksamuel/elastic4s/analysis/Tokenizer.scala
@@ -84,7 +84,7 @@ case class NGramTokenizer(override val name: String,
 
   override def build: XContentBuilder = {
     val b = XContentFactory.jsonBuilder()
-    b.field("type", "nGram")
+    b.field("type", "ngram")
     b.field("min_gram", minGram)
     b.field("max_gram", maxGram)
     if (tokenChars.nonEmpty)

